paper-radio-button.html

```diff
@@ -114,6 +114,7 @@
     },
 
     ready: function() {
+      this.toggles = true;
```

Comment:
Hmm, it is inherited, but I don't want the user to be able to change it, really. If I put it in `properties` it becomes documented and everything, and inspires "change me". No?